pom.xml

```diff
@@ -228,6 +228,11 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.jcraft</groupId>
+            <artifactId>jsch</artifactId>
+            <version>0.1.55</version>

```

Comment:
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: com.jcraft:jsch:jar -> version 0.1.54 vs 0.1.55 @ line 231, column 21. 
This new warning pops up that can break our build potentially can we upgrade the version to the LTS one?